components/dashboard/src/settings/Preferences.tsx

```diff
@@ -73,61 +150,43 @@ export default function Preferences() {
-                        <div className="flex justify-center mt-3">
-                            <img className="w-16 filter-grayscale self-center" src={vscode} />
-                        </div>
-                    </SelectableCard>
-                    <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultDesktopIde === 'code-desktop-insiders'} onClick={() => actuallySetDefaultDesktopIde('code-desktop-insiders')}>
-                        <div className="flex justify-center mt-3">
-                            <img className="w-16 filter-grayscale self-center" src={vscodeInsiders} />
-                        </div>
-                        <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-2 self-center">Insiders</PillLabel>
-                    </SelectableCard>
-                    <SelectableCard className="w-36 h-40" title="IntelliJ IDEA" selected={defaultDesktopIde === 'intellij'} onClick={() => actuallySetDefaultDesktopIde('intellij')}>
-                        <div className="flex justify-center mt-3">
-                            <img className="w-16 filter-grayscale self-center" src={ideaLogo} />
-                        </div>
-                    </SelectableCard>
-                    <SelectableCard className="w-36 h-40" title="GoLand" selected={defaultDesktopIde === 'goland'} onClick={() => actuallySetDefaultDesktopIde('goland')}>
-                        <div className="flex justify-center mt-3">
-                            <img className="w-16 filter-grayscale self-center" src={golandLogo} />
-                        </div>
-                    </SelectableCard>
+                    {
+                        Object.entries(idePreferences.options).filter(([_, x]) => x.type === "desktop").map(([id, option]) => {
+                            const selected = defaultDesktopIde === id;
+                            const onSelect = () => actuallySetDefaultDesktopIde(id);
+                            return renderIdeOption(option, selected, onSelect);
+                        })
+                    }
                 </div>
-                <InfoBox className="my-5 max-w-2xl">While in beta, when you open a workspace using a JetBrains IDE you will need to use the following password: <CodeText>gitpod</CodeText></InfoBox>
-                <p className="text-left w-full text-gray-500">
-                    The <strong>JetBrains desktop IDEs</strong> are currently in beta. <a href="https://github.com/gitpod-io/gitpod/issues/6576" target="gitpod-feedback-issue" rel="noopener" className="gp-link">Send feedback</a> · <a href="https://www.gitpod.io/docs/integrations/jetbrains" target="_blank" rel="noopener" className="gp-link">Documentation</a>
-                </p>
+
+                {idePreferences.desktopIdeInfobox ? <InfoBox className="my-5 max-w-2xl"><div dangerouslySetInnerHTML={{ __html: idePreferences.desktopIdeInfobox }}></div></InfoBox> : <></>}
```

Comment:
I wonder is there any security concerns of using `dangerouslySetInnerHTML` with remotely provided data? Or everything under our control and cannot be hijacket? 

@csweichel Could you have a look, raw HTML will be provided form the server eventually as far as I understand.